packages/ens-referrals/src/referrer.ts

```diff
@@ -0,0 +1,56 @@
+import { type Address, bytesToHex, type Hex, hexToBytes, pad, zeroAddress } from "viem";
+
+/**
+ * Encoded Referrer
+ *
+ * Guaranteed to be a string representation of 32-bytes.
+ */
+export type EncodedReferrer = Hex;
+
+/**
+ * Build an {@link EncodedReferrer} value from a given {@link Address}.
+ */
+export function buildEncodedReferrer(address: Address): EncodedReferrer {
+  const normalizedAddress = address.toLowerCase() as Address;
```

Comment:
```suggestion
  const lowercaseAddress = address.toLowerCase() as Address;
```

Motivation: This library is for use outside of ENSNode and our internal terminology.